bundle/compliance/lib/output_validations/output_validations.rego

```diff
@@ -21,20 +21,16 @@ validate_k8s_metadata(metadata) {
 	metadata.benchmark.name
 	metadata.benchmark.version
 	metadata.benchmark.id
+}
+
+validate_k8s_metadata(metadata) {
```

Comment:
We've had this divide because we did not implement the metadata for EKS.
Now that we have the metadata both for EKS and K8S, I think that we can just remove this divide and use a singular function.
Do you see any value in keeping it?

Reply:
I do see value in it since it is:
a. More readable - we have a function for each benchmark (which makes it a bit easier for the reader).
b. Flexible - we can easily add one field in one benchmark without harming the other.


That being said, I don't think It's that important - if you insist on it, I can remove